File: ReportsAndTranscripts/GenerateReports/GenerateReports.java
package ReportsAndTranscripts.GenerateReports;

import com.mashape.unirest.http.*;
import com.mashape.unirest.http.exceptions.UnirestException;

public class GenerateReports {
    public static void main(String[] args) throws UnirestException{
        HttpResponse<String> response = Unirest
                .get("https://reports.apexchat.com/Services/ApexChatService.svc/queueduserreports/17687?format=json")
                .header("apexchat-company", "CompanyName")
                .header("apexchat-username", "Username")
                .header("apexchat-password", "Password")
                .asString();
        System.out.println(response.getBody());
    }
}

File: ReportsAndTranscripts/GenerateReports/GenerateReports.cs
using System;
using RestSharp;

namespace GenerateReports
{
    class GenerateReports
    {
        static void Main(string[] args)
        {
            var client = new RestClient("https://reports.apexchat.com/Services/ApexChatService.svc/queueduserreports/17687?format=json");
            var request = new RestRequest(Method.GET);
            request.AddHeader("apexchat-company", "CompanyName");
            request.AddHeader("apexchat-username", "Username");
            request.AddHeader("apexchat-password", "Password");
            IRestResponse response = client.Execute(request);
            Console.WriteLine(response.Content);
        }
    }
}
